tools/Customization/DevHome.Customization/Views/DevDriveInsightsView.xaml

```diff
@@ -15,39 +17,55 @@
+                                <ic:DataTriggerBehavior Binding="{x:Bind IsDevDriveTrusted}" Value="True">
+                                    <ic:ChangePropertyAction PropertyName="Foreground" Value="{ThemeResource SystemFillColorSuccessBrush}"/>
+                                </ic:DataTriggerBehavior>
+                            </i:Interaction.Behaviors>
+                        </TextBlock>
+                    </ctControls:SettingsExpander.Description>   
+                    <ctControls:SettingsExpander.HeaderIcon>
                         <FontIcon Glyph="&#xEDA2;" FontSize="25" FontFamily="{ThemeResource AmcFluentIcons}"/>
-                    </ctControls:SettingsCard.HeaderIcon>
+                    </ctControls:SettingsExpander.HeaderIcon>
                     <Grid Grid.Column="2">
                         <Grid.RowDefinitions>
                             <RowDefinition Height="Auto"  />
                             <RowDefinition Height="Auto" />
                             <RowDefinition Height="Auto" />
                         </Grid.RowDefinitions>
                         <TextBlock Style="{StaticResource CardBodyStackPanelAltTextBlockStyle}" HorizontalAlignment="Right" Margin="0 0 115 5">
-                                <Run Text="{Binding Path=DevDriveSizeText, Mode=OneWay}" />
+                            <Run Text="{Binding Path=DevDriveSizeText, Mode=OneWay}" />
                         </TextBlock>
                         <ProgressBar Grid.Row="1" Width="200" Value="{x:Bind DevDriveFillPercentage, Mode=OneWay}" HorizontalAlignment="Right" />
                         <Grid Grid.Row="2">
                             <TextBlock Style="{StaticResource CardBodyStackPanelAltTextBlockStyle}" HorizontalAlignment="Right" Margin="0 0 120 0">
-                                    <Run Text="{Binding Path=DevDriveUsedSizeText, Mode=OneWay}" />
+                                <Run Text="{Binding Path=DevDriveUsedSizeText, Mode=OneWay}" />
                             </TextBlock>
                             <TextBlock Style="{StaticResource CardBodyStackPanelAltTextBlockStyle}" HorizontalAlignment="Right">
-                                    <Run Text="{Binding Path=DevDriveFreeSizeText, Mode=OneWay}" />
+                                <Run Text="{Binding Path=DevDriveFreeSizeText, Mode=OneWay}" />
                             </TextBlock>
                         </Grid>
                     </Grid>
-                </ctControls:SettingsCard>
+                    <ctControls:SettingsExpander.Items>
+                        <ctControls:SettingsCard Visibility="{Binding Path=IsDevDriveTrusted, Mode=OneWay, Converter={StaticResource CollapsedWhenTrueBoolToVisibilityConverter}}" Header="{Binding Path=UntrustedDevDriveText, Mode=OneWay}" Description="{Binding Path=UntrustedDevDriveDescription, Mode=OneWay}">
+                            <Button x:Name="UACLauncher" Command="{Binding MakeDevDriveTrustedCommand}" Width="120">

```

Comment:
Use MinWidth so the button can expand to fit scaled text, otherwise you'll have an accessibility bug.
```suggestion
                            <Button x:Name="UACLauncher" Command="{Binding MakeDevDriveTrustedCommand}" MinWidth="120">
```